projects/plugins/crm/includes/ZeroBSCRM.MetaBoxes3.Quotes.php

```diff
@@ -172,42 +172,33 @@ public function html( $quote, $metabox ) {
                     // we pass the hash along the chain here too :)
                     echo '<input type="hidden" name="zbscq_hash" id="zbscq_hash" value="' . (isset($quote['hash']) ? esc_attr( $quote['hash'] ) : '') . '" />';
                 ?>
-                <style>
-                    @media all and (max-width:699px){
-                        table.wh-metatab{
-                            min-width:100% !important;
-                        }
-                    }  
-                </style>
-                <table class="form-table wh-metatab wptbp" id="wptbpMetaBoxMainItem">
-
+					<div>
+						<div class="jpcrm-form-grid" id="wptbpMetaBoxMainItem">
                     <?php 
 
                     // DAL3 only show after saved, easier
                     if (!empty($quoteID) && $quoteID > 0){
-
-                        // QUOTE ID is seperate / unchangable
-                        ?><tr class="wh-large"><th><label><?php esc_html_e('Quote (ID)',"zero-bs-crm");?>:</label></th>
-                        <td>
-                            <div class="zbs-prominent"><?php 
-
-                            if (empty($quoteID)) $quoteID = zeroBSCRM_getNextQuoteID();
-
-                            echo esc_html( $quoteID );
-
-                            ?><input type="hidden" name="zbsquoteid" value="<?php echo esc_attr( $quoteID ); ?>" /></div>
-                        </td></tr><?php
+					?>
+								<div class="jpcrm-form-group jpcrm-form-group-span-2">
+									<label class="jpcrm-form-label"><?php esc_html_e( 'Quote (ID)', 'zero-bs-crm' ); ?>:</label>
+								<?php
+							if ( empty( $quoteID ) ) { //phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
+								$quoteID = zeroBSCRM_getNextQuoteID(); //phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
+							}
```

Comment:
Legacy code note: correct me if I'm wrong, but I think this condition never is possible given the parent condition?